sql/hive-thriftserver/src/test/scala/org/apache/spark/sql/hive/thriftserver/ThriftServerQueryTestSuite.scala

```diff
@@ -53,6 +54,15 @@ import org.apache.spark.sql.types._
  */
 class ThriftServerQueryTestSuite extends SQLQueryTestSuite with SharedThriftServer {
 
+  override protected def testFile(fileName: String): String = {
+    val url = Thread.currentThread().getContextClassLoader.getResource(fileName)
```

Comment:
oh, I totally missed the failures on Maven env... Thanks, @dongjoon-hyun .
Btw, why the tests passed on sbt env only?

Reply:
SBT test framework seems to get the correct resource path instead of the embedded resource path.

Reply:
hm, I see. Thanks.

Reply:
@dongjoon-hyun Thanks for your fix. If we could copy files from class path to temp file. This indicates that the resource file exists in the jar package. So why Maven can't find the path?

Reply:
Btw, could we add a judge about env(e.g sbt) to avoid copying files ?